eng/versioning/version_client.txt

```diff
@@ -445,7 +445,7 @@ com.azure.tools:azure-sdk-build-tool;1.0.0;1.1.0-beta.1
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
-unreleased_com.azure:azure-core-test;1.22.0-beta.1
+unreleased_com.azure:azure-core-test;1.22.0
```

Comment:
Is this how we usually do this for unreleased dependencies? Will we update the libraries that use it in the "increment versions" PR?

Reply:
This needs to be updated otherwise the libraries using the unreleased dependency will try to use `1.22.0-beta.1` which won't exist. In the version update PR this is manually updated to have libraries using this switch back to the normal dependency usage. Longer term should look into updating the dependency update script to handle this for us.